pom.xml

```diff
@@ -43,6 +44,8 @@
                 </executions>
             </plugin>
           </plugins>
+               </pluginManagement>
```

Comment:
Is this change to pom.xml really needed?

Reply:
Lo que realmente queria subir eran los escenarios no el archivo Pom, me equivoque. Ahora subo los escenarios